imports/plugins/included/product-detail-simple/client/components/childVariant.js

```diff
@@ -50,14 +50,14 @@ class ChildVariant extends Component {
     if (inventoryManagement && this.props.variant.inventoryQuantity <= 0) {
       if (inventoryPolicy) {
         return (
-          <span className="variant-qty-sold-out badge badge-danger">
+          <span className="variant-qty-sold-out badge badge-danger" style={{ marginLeft: "5px" }}>
```

Comment:
No inline styles. This needs to go into a LESS file somewhere. Probably one of these files in this folder `/imports/plugins/included/default-theme/client/styles/products`